nexus/src/db/datastore.rs

```diff
@@ -31,7 +31,29 @@ use crate::authz::ApiResourceError;
 use crate::context::OpContext;
 use crate::db::fixed_data::role_assignment_builtin::BUILTIN_ROLE_ASSIGNMENTS;
 use crate::db::fixed_data::role_builtin::BUILTIN_ROLES;
-use crate::db::fixed_data::silo_builtin::SILO_ID;
+use crate::db::fixed_data::silo::{DEFAULT_SILO, SILO_ID};
+use crate::db::lookup::LookupPath;
+use crate::db::{
+    self,
+    error::{public_error_from_diesel_pool, ErrorHandler, TransactionError},
+    model::{
```

Comment:
Should we just use `model::*` here?

Reply:
In other files I like importing `model` and then using the models qualified with `model::`. However in this file there isn't much opportunity for confusion.

Reply:
I don't have much opinion.  I'd be happy to use `model::` here as well.  But I'm not going to do it in this change -- it'd be a lot of extra delta.